mlir/include/mlir/Dialect/Linalg/TransformOps/LinalgTransformOps.td

```diff
@@ -1899,7 +1897,17 @@ def TileUsingForOp : Op<Transform_Dialect, "structured.tile_using_for",
                       $scalableSizes)>,
   ];
 
-  let hasCustomAssemblyFormat = 1;
+  let assemblyFormat = [{
+    $target oilist(
```

Comment:
No need for `oilist` unless there is an actual list inside it.

Reply:
right. my bad. learned that in the last PR and didn't pay attention here

Reply:
hmm. so actually `oilist` seems to still serve two purposes here. One is that it makes the `vector_sizes` keyword optional. Second is that it will elide `vector_sizes` when the list is empty.  If I change to something like 
```
  let assemblyFormat = [{
    $target
      (`vector_sizes` custom<DynamicIndexList>(
        $vector_sizes,
        $static_vector_sizes,
        $scalable_sizes)^)?
    attr-dict
    `:` type($target)(`,`type($vector_sizes)^)? 
  }];

```
then this still handles the first thing, but not the second. any suggestions?

Reply:
Should I leave as is there another way without it that will keep the current behavior?

Reply:
will do